src/Compilers/CSharp/Portable/CodeGen/EmitStatement.cs

```diff
@@ -1092,7 +1096,22 @@ private void EmitSwitchStatement(BoundSwitchStatement switchStatement)
                         var value = boundLabel.ConstantValueOpt;
                         Debug.Assert(value != null
                             && SwitchConstantValueHelper.IsValidSwitchCaseLabelConstant(value));
-                        labelsBuilder.Add(new KeyValuePair<ConstantValue, object>(value, label));
+
+                        // in debug we should be able to set breakpoints on individual case lables, 
```

Comment:
I don't know what this comment refers to. We do not support breakpoints on case labels, and there is no `nop` instruction between them as there would need to be in order to do so.